### PR TITLE
Review comments

### PR DESCRIPTION
**Pull request by reshdas**
Addressed Satya/Mankamana/Rafal's comments.
Added figure for LBW Ext Community.

### CODE REVIEW

File: draft-ietf-idr-link-bandwidth.xml [hunk 1]
```diff
@@ -170,26 +196,34 @@
...
       </section>
 
       <section anchor="Re-advertisement" title="Re-advertisement Procedures">
         <section anchor="NextHopSelf"
                  title="Re-advertisement with Next hop Self">
           <t>When a BGP speaker re-advertises a route with the Link Bandwidth
           Extended Community and sets the next hop to itself, it SHOULD follow
-          the same procedures as outlined in <xref target="Originator"/>.</t>
+          the same procedures as outlined in <xref target="Originator"/>. If
+          the export policy does not change the Link Bandwidth Extended
```
Comment (kalirajv) [inline draft-ietf-idr-link-bandwidth.xml:223]: keepint this in a separate <t></t> may be better. suggestion:

</t> In the absence of any import, export policies that alter the LBW, a LBW extended community received (if any) on the route is re-advertised as is, following regular BGP procedures. </t>

File: draft-ietf-idr-link-bandwidth.xml [hunk 2]
```diff
@@ -170,26 +196,34 @@
         egress processing to adj-RIB-out only or in ingress processing in
         which case link bandwidth community is present in Local-RIB.</t>
 
-        <t> Note: Implementation may provide configuration option (knob) to
-        allow sending of non-transitive link bandwidth communities to external
-        BGP sessions.</t>
+        <t>Note: Implementation may provide configuration option (knob) to
+        allow sending of non-transitive link bandwidth extended community
```
Comment (kalirajv) [inline draft-ietf-idr-link-bandwidth.xml:200]: "to allow sending non-transitive link bandwidth extended community on external BGP sessions."

and MAY caps?

File: draft-ietf-idr-link-bandwidth.xml [hunk 3]
```diff
@@ -147,12 +147,38 @@
...
+         to indicate transitive/non-transitive.
+
+ SubType: 1-octet field MUST be set to 0x04 
+          to indicate 'Link-Bandwidth'.
+
+ Global Administrator sub-field: 
+          2-octet represent the Autonomous System.
+
+ Local Administrator sub-field: 
+          This bandwidth value is encoded as 4 octets
```
Comment (kalirajv) [inline draft-ietf-idr-link-bandwidth.xml:178]: Bandwidth value (bytes per sec) encoded as 4 octets
in IEEE floating point format.

File: draft-ietf-idr-link-bandwidth.xml [hunk 4]
```diff
@@ -259,13 +259,35 @@
...
+
+      <t>An old sender/receiver is a BGP speakers either use procedures upto
+      draft
+      (https://datatracker.ietf.org/doc/html/draft-ietf-idr-link-bandwidth-08)
+      or undocumented behavior for link bandwidth extended community.</t>
+
+      <t>A new sender/receiver is a BGP speaker that implements procedures
+      specified in this document. </t>
+
+      <t>Old receivers will not be able to interpret or utilize the new
```
Comment (kalirajv) [inline draft-ietf-idr-link-bandwidth.xml:278]: Old receives may not be able to ..

Comment (kalirajv) [reply draft-ietf-idr-link-bandwidth.xml:278]: ok, u mention 'new procedures'. if u remove the new, then it is a 'may'.

File: draft-ietf-idr-link-bandwidth.xml [hunk 5]
```diff
@@ -259,13 +259,35 @@
...
+      procedures described from draft-ietf-idr-link-bandwidth-09, potentially
+      leading to inconsistencies in bandwidth management. The procedures
+      defined in this document does not cover old receivers.</t>
```
Comment (kalirajv) [inline draft-ietf-idr-link-bandwidth.xml:281]: <t> Receiving speaker needs to be upgraded to support the procedures defined in this document to provide full interop with both transitive and non-transitive versions of LBW. In order to keep changes to procedures simple, it is not a goal to provide interop between old Receiver and new Sender. </t>

File: draft-ietf-idr-link-bandwidth.xml (hunk 1 above)
Comment (reshdas) [reply draft-ietf-idr-link-bandwidth.xml]: Hi Kali, 
Thanks for reviewing the document. I have made this change. Please review.

File: draft-ietf-idr-link-bandwidth.xml (hunk 2 above)
Comment (reshdas) [reply draft-ietf-idr-link-bandwidth.xml]: I have made this change. Please review.

File: draft-ietf-idr-link-bandwidth.xml (hunk 3 above)
Comment (reshdas) [reply draft-ietf-idr-link-bandwidth.xml]: I have made this change. Please review.

File: draft-ietf-idr-link-bandwidth.xml (hunk 4 above)
Comment (reshdas) [reply draft-ietf-idr-link-bandwidth.xml]: I have made this change. Please review.

File: draft-ietf-idr-link-bandwidth.xml (hunk 5 above)
Comment (reshdas) [reply draft-ietf-idr-link-bandwidth.xml]: Removed the whole paragraph and added this suggested lines. Please check.

**Review by kalirajv — APPROVED** [review]

**Pull request merged**